File: 0334-increasing-triplet-subsequence/0334-increasing-triplet-subsequence.cpp
class Solution {
public:
    bool increasingTriplet(vector<int>& nums) {
        if(nums.size() < 3) {
            return false;
        }
        int i = INT_MAX, j = INT_MAX;
        for(int l = 0; l < nums.size(); l++) {
            if(nums[l] <= i) {
                i = nums[l];
            } else if(nums[l] <= j) {
                j = nums[l];
            } else {
                return true;
            }
        }
        return false;
    }
};

File: 0334-increasing-triplet-subsequence/0334-increasing-triplet-subsequence.py
class Solution(object):
    def increasingTriplet(self, nums):
        """
        :type nums: List[int]
        :rtype: bool
        """
        if len(nums) < 3:
            return False
        i = sys.maxsize
        j = sys.maxsize
        for l in range(len(nums)):
            if nums[l] <= i:
                i = nums[l]
            elif nums[l] <= j:
                j = nums[l]
            else:
                return True
        return False
        